cli/cmd/user/add.go

```diff
@@ -0,0 +1,78 @@
+package user
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/cmd/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func AddCmd(cfg *config.Config) *cobra.Command {
+	var orgName string
+	var projectName string
+	var email string
+	var role string
+
+	addCmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if orgName == "" {
+				orgName = cfg.Org
+			}
+
+			client, err := cmdutil.Client(cfg)
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			cmdutil.SelectPromptIfEmpty(&role, "Please enter the user's role.", []string{"admin", "viewer"}, "")
```

Comment:
Shorten prompt to `Select role` (no dot)

Reply:
For org-level access, there's also `collaborator`. It's actually also there for projects, so you could just show all three in both cases if it's easier.